From 24a08c158c9bab495093b9ae582d5140456c8459 Mon Sep 17 00:00:00 2001
From: jakenjarvis <jaken.jarvis@gmail.com>
Date: Mon, 17 Sep 2018 18:21:36 +0900
Subject: [PATCH] test2

---
 MSBuild.ILMerge.Task.targets | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/MSBuild.ILMerge.Task.targets b/MSBuild.ILMerge.Task.targets
index aab770f..dd4aa01 100644
--- a/MSBuild.ILMerge.Task.targets
+++ b/MSBuild.ILMerge.Task.targets
@@ -32,7 +32,7 @@
     <Message Text="Transitive merge" Importance="high" Condition="$(ILMergeTransitive) == 'true'" />
 
     <!-- all copy local assemblies referenced from this project that go to the executable except the main one-->
-    <CreateItem Include="@(ReferencePath)" Condition=" '%(CopyLocal)' == 'true' ">
+    <CreateItem Include="@(ReferencePath)" Condition=" '%(ReferencePath.CopyLocal)' == 'true' ">
       <Output TaskParameter="Include"  ItemName="MergedAssemblies"/>
     </CreateItem>
 
@@ -42,7 +42,7 @@
     </CreateItem>
     
     <!-- all assemblies that doesn't so we use their directories as library path -->
-    <CreateItem Include="@(ReferencePath)" Condition=" '%(CopyLocal)' == 'false' ">
+    <CreateItem Include="@(ReferencePath)" Condition=" '%(ReferencePath.CopyLocal)' == 'false' ">
       <Output TaskParameter="Include" ItemName="UnmergedAssemblies"/>
     </CreateItem>
 
-- 
2.18.0.windows.1

