diff -uprN ./PreventReboot_a/packages/MSBuild.ILMerge.Task.1.0.5/build/MSBuild.ILMerge.Task.targets ./PreventReboot_b/packages/MSBuild.ILMerge.Task.1.0.5/build/MSBuild.ILMerge.Task.targets
--- ./PreventReboot_a/packages/MSBuild.ILMerge.Task.1.0.5/build/MSBuild.ILMerge.Task.targets	2016-07-11 13:39:00.000000000 +0900
+++ ./PreventReboot_b/packages/MSBuild.ILMerge.Task.1.0.5/build/MSBuild.ILMerge.Task.targets	2018-09-17 21:34:05.000000000 +0900
@@ -32,7 +32,7 @@
     <Message Text="Transitive merge" Importance="high" Condition="$(ILMergeTransitive) == 'true'" />
 
     <!-- all copy local assemblies referenced from this project that go to the executable except the main one-->
-    <CreateItem Include="@(ReferencePath)" Condition=" '%(CopyLocal)' == 'true' ">
+    <CreateItem Include="@(ReferencePath)" Condition=" '%(ReferencePath.CopyLocal)' == 'true' ">
       <Output TaskParameter="Include"  ItemName="MergedAssemblies"/>
     </CreateItem>
 
@@ -42,7 +42,7 @@
     </CreateItem>
     
     <!-- all assemblies that doesn't so we use their directories as library path -->
-    <CreateItem Include="@(ReferencePath)" Condition=" '%(CopyLocal)' == 'false' ">
+    <CreateItem Include="@(ReferencePath)" Condition=" '%(ReferencePath.CopyLocal)' == 'false' ">
       <Output TaskParameter="Include" ItemName="UnmergedAssemblies"/>
     </CreateItem>
 
